packages/react/src/components/tabs/Chip/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Flex from "../../layout/Flex";
+
+export type Props = React.PropsWithChildren<{
+  /**
+   * An optional callback that will be called when the active tab changes.
+   */
+  onTabChange?: (activeIndex: number) => void;
+  /**
+   * The tab index to mark as active when rendering for the first time.
+   * If omitted, then initially no tabs will be selected.
+   */
+  initialActiveIndex?: number;
+}>;
+type ItemProps = {
+  active: boolean;
+};
+
+const Item = styled(Flex).attrs({
+  flex: 1,
+  justifyContent: "center",
+  alignItems: "center",
+})<ItemProps>`
+  cursor: pointer;
+  padding: 8px 12px 8px 12px;
+  border-radius: ${(p) => p.theme.radii[2]}px;
+  color: ${(p) => p.theme.colors.palette.neutral.c100};
+  background-color: ${(p) => (p.active ? p.theme.colors.palette.primary.c30 : "unset")};
+`;
+
+export default function BarTabs({ children, onTabChange, initialActiveIndex }: Props): JSX.Element {
+  const [activeIndex, setActiveIndex] = useState(initialActiveIndex);
+  return (
+    <Flex justifyContent="space-between" flex={1} columnGap={5}>
+      {React.Children.toArray(children).map((child, index) => (
```

Comment:
not the biggest fan of this method to loop through content :thinking: 
why not use a tabs list in props with keys and content and text styling in the items directly.
It should ease the actual integration in the end with less copy paste of template 
```js
<BarTabs>
<Text variant="h2">1</Text>
<Text variant="h2">2</Text>
<Text variant="h2">3</Text>
</BarTabs>

-->
<BarTabs tabs={[
{key: 1, label: "1"},
{key: 2, label: "2"},
{key: 3, label: "3"},
]}>
```
its a little less prone to style interpretation imo

Reply:
I've re-discussed this with @jelbaz-ledger and if it's not a big blocker for you. We'd rather keep it this way, as it makes it simpler for customizing the component and display other things besides text.
Would it be ok for you?